README.md

```diff
@@ -304,90 +320,85 @@ Any number of downloaders of any type may be configured.
+| qbit.user      | `DN_QBIT_0_USER`      | No Default. Provide URL, user and pass if you use Qbit        |
+| qbit.pass      | `DN_QBIT_0_PASS`      | No Default. Provide URL, user and pass if you use Qbit        |
+| qbit.http_user | `DN_QBIT_0_HTTP_USER` | Provide this username if Qbit is behind basic auth (uncommon) |
+| qbit.http_pass | `DN_QBIT_0_HTTP_PASS` | Provide this password if Qbit is behind basic auth (uncommon) |
 
 #### SABnzbd
 
-|Config Name|Variable Name|Note|
-|---|---|---|
-sabnzbd.name|`DN_SABNZBD_0_NAME`|No Default. Setting a name enables service checks|
-sabnzbd.url|`DN_SABNZBD_0_URL`|No Default. Something like: `http://localhost:8080/sabnzbd`|
-sabnzbd.api_key|`DN_SABNZBD_0_API_KEY`|No Default. Provide URL and API key if you use SABnzbd|
+| Config Name     | Variable Name          | Note                                                        |
+| --------------- | ---------------------- | ----------------------------------------------------------- |
+| sabnzbd.name    | `DN_SABNZBD_0_NAME`    | No Default. Setting a name enables service checks           |
+| sabnzbd.url     | `DN_SABNZBD_0_URL`     | No Default. Something like: `http://localhost:8080/sabnzbd` |
+| sabnzbd.api_key | `DN_SABNZBD_0_API_KEY` | No Default. Provide URL and API key if you use SABnzbd      |
 
 #### Deluge
 
-|Config Name|Variable Name|Note|
-|---|---|---|
-deluge.name|`DN_DELUGE_0_NAME`|No Default. Setting a name enables service checks|
-deluge.url|`DN_DELUGE_0_URL`|No Default. Something like: `http://localhost:8080`|
-deluge.password|`DN_DELUGE_0_PASSWORD`|No Default. Provide URL and password key if you use Deluge|
-deluge.http_user|`DN_DELUGE_0_HTTP_USER`|Provide this username if Deluge is behind basic auth (uncommon)|
-deluge.http_pass|`DN_DELUGE_0_HTTP_PASS`|Provide this password if Deluge is behind basic auth (uncommon)|
+| Config Name      | Variable Name           | Note                                                            |
+| ---------------- | ----------------------- | --------------------------------------------------------------- |
+| deluge.name      | `DN_DELUGE_0_NAME`      | No Default. Setting a name enables service checks               |
+| deluge.url       | `DN_DELUGE_0_URL`       | No Default. Something like: `http://localhost:8080`             |
+| deluge.password  | `DN_DELUGE_0_PASSWORD`  | No Default. Provide URL and password key if you use Deluge      |
+| deluge.http_user | `DN_DELUGE_0_HTTP_USER` | Provide this username if Deluge is behind basic auth (uncommon) |
+| deluge.http_pass | `DN_DELUGE_0_HTTP_PASS` | Provide this password if Deluge is behind basic auth (uncommon) |
 
 ### Plex
 
-This application can also send Plex sessions to Notfiarr so you can receive
-notifications when users interact with your server. This has three different features:
+This application can also send Plex sessions to Notifiarr so you can receive notifications when users interact with your server. This has three different features:
```

Comment:
I'm confused why you made all the lines longer. I keep them short so I can read them on my tiny ass screen.

Reply:
and if you merge this, I'm just going to wrap them again next time I edit the readme.

Reply:
GH and everything autowraps for it to be viewable on small screens?

Reply:
I don't edit markdown files on Github's website.